### DISCUSSION TITLE
How to restore Activity Bar to Default

### DISCUSSION THREAD

**Discussion by tabedzki** [microsoft/vscode-discussions#872] [Extension Development QnA] [answered]
Hi 
I accidentally dragged and dropped my explorer activity bar into the git icon, now the folders doesn't work like it should. 
I dragged things back to have a first "tile" which would have  "Open Editors, Folder, Outline Timeline" like it did originally, except when I do that, it now calls the entire tile Open Editors instead of the Folders. 

I would like help getting this back to the original style since the hotkeys are now broken in this state since it has the wrong header.

Thank you in advance

**Comment by isidorn** [comment, marked as answer]
Just drag and drop. And ask these questions on Stack Overflow or to ChatGPT.
This is a forum for VS Code extension authors. Thank you

**Reply by tabedzki** [reply]
Drag and drop wasn't working for me unfortunately but ChatGPT did provide the correct answer. I got confused where to post because the repo name is `microsoft/vscode-discussions`
